variables.tf

```diff
@@ -32,6 +32,11 @@ variable "metal_project_id" {
   description = "Your Equinix Metal Project ID"
 }
 
+variable "metal_vlan" {
```

Comment:
We'll want to allow for existing VLANs to be plumbed in. The network module already supports that.